engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/transformer/FlowElementInstantiationTransformer.java

```diff
@@ -7,6 +7,8 @@
  */
 package io.camunda.zeebe.engine.processing.deployment.model.transformer;
 
+import static io.camunda.zeebe.model.bpmn.validation.zeebe.FlowElementValidator.NON_EXECUTABLE_ELEMENT_TYPES;
```

Comment:
As mentioned above, instead of importing the `NON_EXECUTABLE_ELEMENT_TYPES` HashSet, let's duplicate this data into this class.